api/src/data/migrations/052.update-data-injection-sources-table.ts

```diff
@@ -0,0 +1,15 @@
+import { Knex } from "knex";
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable("data_injection_sources", (table) => {
+    table.string("identifier_column_name", 250).notNullable().alter();
```

Comment:
Generally, if you are making an existing column non-nullable, you need to set it's data first.
Unless this column guaranteed to be empty?